distribution/lib/Standard/Base/0.0.0-dev/src/Internal/Data_Read_Helpers.enso

```diff
@@ -28,8 +33,22 @@ looks_like_uri path:Text -> Boolean =
    A common implementation for fetching a resource and decoding it,
    following encountered data links.
 fetch_following_data_links (uri:URI) (method:HTTP_Method = HTTP_Method.Get) (headers:Vector = []) format =
-    response = HTTP.fetch uri method headers
-    decode_http_response_following_data_links response format
+    fetch_and_decode =
+        response = HTTP.fetch uri method headers
+        decode_http_response_following_data_links response format
+
+    error_handler attempt =
+        caught_error ->
+            case method of
+                HTTP_Method.Get ->
+                    if attempt > 2 then Error.throw caught_error else
+                        sleep_time = Math.min (100 * (2 ^ (attempt - 1))) 5000
```

Comment:
```suggestion
                        sleep_time = [100, 200, 400]. get attempt
```

Is probably clearer (I know I sent the above), but also needs to `attempt` not `attempt-1`.